File: Java programming/problems on string manipulation/problem103.java
/*
103.Write a program which accept string from user and toggle the case. 
	
		Input :   “Marvellous Multi OS” 
		Output :   mARVELLOUS mULTI os 
	
		Author : Milind Nalawade
		
		Date : 15/07/2021
		
		language : Java 14.0.2
		
*/

import java.lang.*;
import java.util.*;

class Logic
{
	public static void Convert(String arr)
	{
		int ln=arr.length();
		char str[] =new char[arr.length()];
		for (int i=0;i<ln;i++)
		{
			str[i]=arr.charAt(i);
			if(str[i]>=65&&str[i]<=90)
			{
				char temp=Character.toLowerCase(str[i]);
				str[i]=temp;
			}
			else if(str[i]>=97&&str[i]<=122)
			{
				char temp=Character.toUpperCase(str[i]);
				str[i]=temp;
			}
		}
		System.out.println();
		for(int j=0;j<ln;j++)
		{
			System.out.print(str[j]);
		}
		System.out.println();
	}
}

class problem103 extends Logic
{
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.print("\n Enter your string : ");
		String str=sobj.nextLine();
		Convert(str);
	}
	
}		

File: C Programming/problems on strings maipulation/problem103.c
/*

103.Write a program which accept string from user and toggle the case. 

	Input :   “Marvellous Multi OS” 
	Output :   mARVELLOUS mULTI os 
		
		Author : Milind Nalawade
		
		Date : 15/07/2021
		
		language : C programming

*/

#include<stdio.h>
#include<string.h>

void Covert(char* arr)
{
	for (int i=0;i<strlen(arr);i++)
	{
		if((arr[i]>=97)&&(arr[i]<=122))
		{
			char temp=arr[i]-32;
			arr[i]=temp;
		}
		else if(arr[i]>=65&&arr[i]<=90)
		{
			char ch=arr[i]+32;
			arr[i]=ch;
		}
	}	
	printf("\n ");
	printf(" %s",arr);
	printf("\n");
}

//Entry point function

int main()
{	
	char arr[20];
	printf("\n Enter your string : ");
	scanf("%[^'\n']s",arr);
	Covert(arr);
	return 0;
}		